generation/localizedstringkit/__init__.py

```diff
@@ -16,35 +16,44 @@
 from localizedstringkit.files import localizable_files
 
 
-SOURCE_CODE_FILE_NAME: str = "source_strings.m"
-BUNDLE_NAME: str = "LocalizedStringKit.bundle"
-
 log = logger.get()
 
 
-def generate_code_strings_file(code_files: List[str]) -> str:
-    """Generate a single code file with all strings.
+def generate_code_strings_file(code_files: List[str]) -> dict[str: str]:
+    """Generate a single code file with all strings per bundle.
 
     :param code_files: The list of file paths to generate the code strings for
 
-    :returns: The path to the temporary source code file with the standard NSLocalizedString calls
+    :returns: A dictionary of bundle name and path to the temporary source code file with the standard NSLocalizedString calls for that bundle
     """
 
-    output_path = tempfile.mktemp(suffix=".m")
-
+    # Get & dedupe localized strings
     localized_strings = detection.strings_in_code_files(code_files)
     localized_strings = list(set(localized_strings))
 
+    bundles: List[str] = []
```

Comment:
You can make `bundles` a set and avoid the `bundle not in bundles` check. 

e.g.

`bundles = {localized_string.bundle for localized_string in localized_strings if localized_string.bundle}`

Reply:
Updated